<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>496</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LUD Media Converter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/penguin.png</normaloff>:/images/penguin.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome to LUD Media Converter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectFileButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Media File</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier 10 Pitch</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="outputFormatGroupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>461</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Output Format</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="androidHDRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Android HD (Galaxy Note 2, S3, Transformer)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="androidQHDRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Android qHD (HTC Sensation, Galaxy S2)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="appleHDRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Apple HD (iPhone 4, iPhone 4S)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="appleFullHDRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>441</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Apple Full HD (iPhone 5, iPad2, iPad 3, iPad 4, iPad Mini)</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="statusLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier 10 Pitch</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Output Format:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>761</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier 10 Pitch</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputFormat">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier 10 Pitch</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Output Format</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>210</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Conversion Status</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>781</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
